src/frontend/shortlist/src/components/preferences/__tests__/PreferenceMultiSelect.test.js

```diff
@@ -0,0 +1,35 @@
+import { describe, it, expect } from "vitest";
+import { mount } from "@vue/test-utils";
+import PreferenceMultiSelect from "../PreferenceMultiSelect.vue";
+
+describe("PreferenceMultiSelect.vue", () => {
+  // Check if component exists
+  it("renders with null props", () => {
+    const componentWrapper = mount(PreferenceMultiSelect, {
+      props: {
+        question: "",
+        modelValue: [[""]],
```

Comment:
I think this one should just be like `[""]` or `["sample1", "sample2"]`, but if this needs to be a list of lists to make it pass that's fine